org.civicrm.reqangular/src/common/decorators/angular-date/datepicker-addon.js

```diff
@@ -1,35 +1,88 @@
-define([], function () {
+/* eslint-env amd */
+
+define([
+  'common/moment'
+], function (moment) {
   'use strict';
 
   /**
    * Decorates the uib-datepicker-popup directive, making its sibling '.input-group-addon'
    * trigger the datepicker
    */
-  return ['$delegate', function ($delegate) {
+  return ['$delegate', '$compile', function ($delegate, $compile) {
     var directive = $delegate[0];
+    var isMobile = document.getElementsByTagName('body')[0].classList.contains('mobile');
     var origCompile = directive.compile;
+    var calendarIcon = '<span ' +
```

Comment:
`calendarIconMarkup`

Reply:
Moved the code to `angular.element`, so kept previous name.